Actions in reference (#216)
avohq/docs: pages/implementation/destinations/managed-destinations.mdx

  />
</center>

> If you choose Avo to manage your destination, you'll still have to include the destination SDK in your application, but the initialization of the library and all event actions: Log Event, Update User Properties, Identify User, Unidentify User, Log Revenue, Log Page View, Update Groups, will be handled by the Avo generated code, so you don't need to think about that.

Reviewer: ```suggestion
If you choose Avo to manage your destination, you'll still have to include the destination SDK in your application, but the initialization of the library and all event actions: `Log Event`, `Update User Properties`, `Identify User`, `Unidentify User`, `Log Revenue`, `Log Page View`, `Update Groups`, will be handled by the Avo generated code, so you don't need to think about that.
```

- Avo will initialize the analytics SDK with the API key you provided in the Avo UI corresponding to the specified environment when you initialize Avo

> - Avo will call the track method of the analytics SDK (e.g. `track(eventName, eventProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Log Event` action is called and will include the event and system properties in the call

Reviewer: ```suggestion
- `Log Event`: Avo will call the track method of the analytics SDK (e.g. `track(eventName, eventProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Log Event` action is called and will include the event and system properties in the call
```

> - Avo will associate the track method with the specified group if the group is provided in the `Log Event` action

Reviewer: ```suggestion
- `Log event with Group`: Avo will associate the track method with the specified group if the event is associated with a group in the `Log Event` action.
```

> - Avo will call the user properties method of the analytics SDK (e.g. `identify(userId, userProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Update User Properties` action is called and will include the user properties in the call

Reviewer: ```suggestion
- `Identify User`: Avo will call the user properties method of the analytics SDK (e.g. `identify(userId, userProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Update User Properties` action is called and will include the user properties in the call.
```

- Avo will call the identify method of the analytics SDK (e.g. `identify(userId)` in the Segment SDK) when an Avo Function corresponding to an event with `Identify User` action is called

> - Avo will call the unidentify method of the analytics SDK (e.g. `identify(null)` in the Segment SDK) when an Avo Function corresponding to an event with `Unidentify User` action is called

Reviewer: ```suggestion
- `Unidentify User`: Avo will call the unidentify method of the analytics SDK (e.g. `identify(null)` in the Segment SDK) when an Avo Function corresponding to an event with `Unidentify User` action is called
```

> - Avo will call the revenue method of the analytics SDK (e.g. `track("revenue", {"revenue": amount})` in the Segment SDK) when an Avo Function corresponding to an event with `Log Revenue` action is called

Reviewer: ```suggestion
- `Log Revenue`: Avo will call the revenue method of the analytics SDK (e.g. `track("revenue", {"revenue": amount})` in the Segment SDK) when an Avo Function corresponding to an event with `Log Revenue` action is called
```

> - Avo will call the page method of the analytics SDK (e.g. `page(pageName, eventProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Log Page View` action is called and will include the event and system properties in the call

Reviewer: ```suggestion
- `Log Page View`: Avo will call the page method of the analytics SDK (e.g. `page(pageName, eventProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Log Page View` action is called and will include the event and system properties in the call
```

> - Avo will associate current user with the specified group when an Avo event with the 'Update Groups' action is triggered

Reviewer: ```suggestion
- `Associate user with Group`: Avo will associate current user with the specified group when an Avo event with the 'Update Groups' action is triggered
```

> - Avo will update specified group metadata when an Avo event with the 'Update Groups' action is triggered

Reviewer: ```suggestion
- `Update Groups Properties`: Avo will update specified group metadata when an Avo event with the 'Update Groups' action is triggered.
```
